### PR TITLE
Added nsp check

### PR DESCRIPTION
**Pull request by SergeyZhukovsky**
It is related to that issue
https://github.com/bbondy/abp-filter-parser-cpp/issues/15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,14 +14,17 @@
...
   },
   "scripts": {
     "test": "make test",
     "build": "make",
     "preinstall": "npm install bloom-filter-cpp && npm install hashset-cpp",
     "install": "node-gyp rebuild",
-    "lint": "./scripts/cpplint.py"
+    "lint": "./scripts/cpplint.py",
+    "checks": "npm run check-security",
+    "check-security": "nsp check"
```
Comment (bbondy) [inline package.json]: This should go in the pre-commit script too  please also inside package.json.

Comment (bbondy) [inline package.json]: nevermind don't do that.

**Comment by bbondy** [comment]
Looks good but please rebase it off of origin master though because it won't merge like this.

**Comment by SergeyZhukovsky** [comment]
Done

**Comment by bbondy** [comment]
:) yes now it is good. Thanks :+1:

**Pull request merged**